### ISSUE TITLE
Port all of Windows Workflow to .NET Core

### ISSUE THREAD

**Issue by terrajobst** [dotnet/corefx#23051]
This would depend on #5766 

**Note: this issue isn't a committment from our side to do so, it merely captures the community request.**

**Comment by 4creators** [comment]
@terrajobst Does this mean that there is a decision made to port both System.Xaml and Workflow to .NET Core or it is like in #5766 case simply capturing the community request to do so?

**Comment by danmosemsft** [comment]
@terrajobst note we already had https://github.com/dotnet/corefx/issues/2394

**Comment by sharwell** [comment]
@terrajobst It would be good to address @4creators' question preemptively on issues you file like this. The answer also isn't clear to me. :smile:

**Comment by terrajobst** [comment]
Thanks for asking. I've clarified in the issue body :-)

**Comment by terrajobst** [comment]
@danmosemsft 

Yep, we couldn't find it in the meeting so I just create one. Will close as duplicate of #2394.
